js/widgets/temperament.js

```diff
@@ -782,14 +782,28 @@ function TemperamentWidget() {
         this.tempRatios = [];
 
         divAppend.addEventListener("click", function (event) {
-            that.performEqualEdit(event);
+            try{
+                that.performEqualEdit(event);
+            } catch{
+                alert("Number of divisions cannot exceed 999")
```

Comment:
We should use the MB error msg mechanism. And please put your string inside _() so it will be translated.
Not sure that the error message is correct. Since we don't know the bounds, we should just say that the number of divisions is too large.